docs/triggers_rules.md

```diff
@@ -104,9 +104,14 @@ As an example, create a rule that calls the hello action whenever a location upd
   $ wsk action update hello hello.js
   ```
 
-3. Create and enable the rule. The three parameters are the name of the rule, the trigger, and the action.
+3. Create the rule. Note that the rule will be enabled upon creation, meaning that it will be immediately available to respond to activations of your trigger. The three parameters are the name of the rule, the trigger, and the action.
```

Comment:
i think will be is proper here, because we're describing the possible future. i.e. this sentence occurs before the example command line invocation.